examples/simple-file-server/README.md

```diff
@@ -0,0 +1,74 @@
+# Simple File Server
+
+## Overview
+
+This guide explains how to put and get a file to/from a UDP server using Ballerina.
+The following figure illustrates a high-level design diagram of the complete use case.
```

Comment:
This example explains how to write a simple file server using UDP. The server only supports sending and receiving files. The following figure illustrates the high-level design diagram of the server.